src/styles/blocks/cataloge.scss

```
@@ -0,0 +1,120 @@
+@import "./../utils/variables";
+
+.cataloge {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  width: 1024px;
+  margin: 0 auto;
+
+  &__card {
+    margin: 0 48px 46px 0;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  padding: 32px 16px 16px;
+  background: $bg-color;
+  border: 1px solid $border-card-elements;
+  border-radius: 5px;
+  width: 200px;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+  
+  &__title {
+    font-family: "Roboto", sans-serif;
+    font-size: $title-size;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin-bottom: 4px;
+  }
+  
+  &__code {
+    font-family: "Roboto-Regular", sans-serif;
+    font-weight: 400;
+    font-size: $secondary-size;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+  
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+  
+  &__number-review {
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+    font-size: $secondary-size;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 18px;
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+  
+  &__price-text {
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+    font-size: $price-text;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+  
+  &__price-count {
+    font-family: "Roboto", sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+  
+  &__button {
+    font-family: "Roboto-Bold", sans-serif;
```

Where did it come from font-family Roboto-Regular?
```suggestion
    font-family: "Roboto", sans-serif;
```